bundles/binding/org.openhab.binding.harmonyhub/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="/org.openhab.io.harmonyhub/target/org.openhab.io.harmonyhub-1.7.0-SNAPSHOT.jar"/>
+	<classpathentry kind="lib" path="/org.openhab.io.harmonyhub/lib/harmony-java-client-1.0.3-all.jar"/>
```

Comment:
I can break it up into sub jars of that would be better, it depends on:
'com.google.guava:guava:17.0'
'com.fasterxml.jackson.core:jackson-core:2.4.2'
'com.fasterxml.jackson.core:jackson-databind:2.4.2'
'com.google.inject:guice:4.0-beta4'
'org.igniterealtime.smack:smack-core:4.0.2'
'org.igniterealtime.smack:smack-tcp:4.0.2'
'org.igniterealtime.smack:smack-debug:4.0.2'
'us.monoid.web:resty:0.3.2'

I'll give it a try, if it ends up being an issue I come back and comment